packages/_internal/stylelint-config-syn/package.json

```diff
@@ -35,6 +35,7 @@
   ],
   "license": "BSD-3-Clause",
   "name": "@synergy-design-system/stylelint-config-syn",
+  "private": true,
```

Comment:
Should we rename the folder `_internal` to `_private` to have a common naming?

Reply:
> Should we rename the folder `_internal` to `_private` to have a common naming?

Propably, I will change it in the branch directly.

Reply:
Done in latest version